mitosheet/mitosheet/step_performers/graph_steps/graph_utils.py

```diff
@@ -80,6 +81,25 @@ def get_graph_title(
     return " ".join(graph_title_components)
 
 
+def get_new_graph_tab_name(graph_data: Dict[str, Dict[str, Any]]) -> str:
+    """
+    Creates the name for the new graph tab sheet using the format
+    graph0, graph1, etc.
+    """
+    all_graph_names = []
+    for graph_data in graph_data.values():
```

Comment:
This is dangerous! You're redefining an exising variable, which means if you try and use the param later on, it'll be defined as one of the values in the graph data dict. Sorry to be a narc here, but this is even worse than value - this _will_ lead to bugs if we ever refactor this function.

What about renaming the param to the function to `graph_data_dict` or something?